test_conformance/api/test_clone_kernel.cpp

```diff
@@ -351,51 +466,459 @@ int test_clone_kernel(cl_device_id deviceID, cl_context context, cl_command_queu
+                                   0, NULL, NULL);
+    test_error(error, "clEnqueueNDRangeKernel failed");
+    error = clFinish(queue);
+
+    test_error(error, "clFinish failed");
+    error = clEnqueueSVMMap(queue, CL_TRUE, CL_MAP_READ | CL_MAP_WRITE,
+                            svmPtr_Kernel, sizeof(cl_int), 0, NULL, NULL);
+    test_error(error, "clEnqueueSVMMap failed");
+    test_assert_error(svmPtr_Kernel[0] == *value,
+                      "clCloneKernel test failed, Failed to verify "
+                      "integer value from SVM pointer. ");
+
+    error = clEnqueueSVMUnmap(queue, svmPtr_Kernel, 0, NULL, NULL);
+    test_error(error, "clEnqueueSVMUnmap failed");
+    error = clFinish(queue);
+    test_error(error, "clFinish failed");
+
+    return TEST_PASS;
+}
+
+int test_svm_exec_info_helper(cl_context context, cl_command_queue queue,
+                              BufPtr* pBuf, cl_int* svmPtr_Kernel,
+                              cl_kernel* srcKernel, cl_int* value)
+{
+    cl_int error;
+
+    error = clEnqueueSVMMap(queue, CL_TRUE, CL_MAP_READ | CL_MAP_WRITE, pBuf,
+                            sizeof(BufPtr), 0, NULL, NULL);
+    test_error(error, "clEnqueueSVMMap failed");
+    error = clFinish(queue);
+    test_error(error, "clFinish failed");
+
+    pBuf->store = svmPtr_Kernel;
+
+    error = clEnqueueSVMUnmap(queue, pBuf, 0, NULL, NULL);
+    test_error(error, "clEnqueueSVMUnmap failed");
+    error = clFinish(queue);
+    test_error(error, "clFinish failed");
+
+    error = clSetKernelArg(*srcKernel, 0, sizeof(cl_int), value);
+    test_error(error, "clSetKernelArg failed");
+    error = clSetKernelArgSVMPointer(*srcKernel, 1, pBuf);
+    test_error(error, "clSetKernelArgSVMPointer failed");
+    error = clSetKernelExecInfo(*srcKernel, CL_KERNEL_EXEC_INFO_SVM_PTRS,
+                                sizeof(pBuf), pBuf);
+    test_error(error, "clSetKernelExecInfo failed");
+
+    error = clFinish(queue);
+    test_error(error, "clFinish failed");
+
+    if (test_exec_enqueue_helper(context, queue, pBuf, svmPtr_Kernel, srcKernel,
+                                 value)
+        != 0)
     {
-        test_error( error, "clCloneKernel test failed. Failed to clone structure type argument." );
-        return -1;
+        test_fail("test_exec_enqueue_helper failed.\n");
     }
 
-    if (((float*)pbufRes)[3] != sa.f)
+    return TEST_PASS;
+}
+
+int test_cloned_kernel_exec_info(cl_device_id deviceID, cl_context context,
+                                 cl_command_queue queue, int num_elements)
+{
+    cl_int error;
+
+    clMemWrapper bufOut;
+    clProgramWrapper program;
+    clKernelWrapper srcKernel;
+
+    cl_int intargs[] = { 1, 2, 3, 4 };
+    cl_device_svm_capabilities svmCaps = 0;
+
+    error = clGetDeviceInfo(deviceID, CL_DEVICE_SVM_CAPABILITIES,
+                            sizeof(svmCaps), &svmCaps, NULL);
+    test_error(error, "Unable to query CL_DEVICE_SVM_CAPABILITIES");
+
+    if (svmCaps != 0)
```

Comment:
I think we should return TEST_SKIPPED_ITSELF in case svmCaps == 0 now there is TEST_PASS